source/Playnite.DesktopApp/Controls/ComboBoxListBase.cs

```diff
@@ -40,12 +78,43 @@ public override void OnApplyTemplate()
             ButtonClearFilter = Template.FindName("PART_ButtonClearFilter", this) as Button;
             TextFilterString = Template.FindName("PART_TextFilterString", this) as TextBlock;
             ItemsPanel = Template.FindName("PART_ItemsPanel", this) as ItemsControl;
+            ButtonCheckedOnly = Template.FindName("PART_CheckedOnly", this) as ToggleButton;
+            TextSearchBox = Template.FindName("PART_SearchBox", this) as SearchBox;
+            SearchOptions = Template.FindName("PART_SearchOptions", this) as Grid;
 
             if (ButtonClearFilter != null)
             {
                 ButtonClearFilter.Click += (_, e) => ClearButtonAction(e);
             }
 
+            if (ButtonCheckedOnly != null)
+            {
+                ButtonCheckedOnly.Click += (_, e) => ButtonCheckedOnlyAction(_, e);
+            }
+
+            if (SearchOptions != null)

```

Comment:
You can't check like that, you should be binding only if the controls are in the template...